resource_processor/shared/logging.py

```diff
@@ -140,8 +140,7 @@ def shell_output_logger(console_output: str, prefix_item: str, logger: logging.L
         logging.debug("Image not present locally, setting log to INFO.")
         logging_level = logging.INFO
 
-    logger.log(logging_level, prefix_item)
-    logger.log(logging_level, console_output)
+    logger.log(logging_level, f"{prefix_item} {console_output}")
```

Comment:
IIRC, this has been done like so because the log has been spread across multiple entries. 
It might not be like that now, but if it is still then this change is going to make it hard to understand.

Reply:
I tested this and the log analytics log were easier to follow with prefix on same line, otherwise the prefix was a separate entry (on its own) to the actual error which isn't useful.